datatypes/src/collections/multi_point_collection.rs

```diff
@@ -1173,31 +1173,26 @@ mod tests {
         )
         .unwrap();
 
-        let pc_expected = MultiPointCollection::from_data(
-            MultiPoint::many(vec![
-                vec![MARBURG_EPSG_900_913, COLOGNE_EPSG_900_913],
-                vec![HAMBURG_EPSG_900_913],
-            ])
-            .unwrap(),
-            vec![
-                TimeInterval::new_unchecked(0, 1),
-                TimeInterval::new_unchecked(1, 2),
-            ],
-            {
-                let mut map = HashMap::new();
-                map.insert("numbers".into(), FeatureData::Float(vec![0., 1.]));
-                map.insert(
-                    "number_nulls".into(),
-                    FeatureData::NullableFloat(vec![Some(0.), None]),
-                );
-                map
-            },
-        )
+        let expected = MultiPoint::many(vec![
```

Comment:
You have a point collection with data as `pc`, but you do nothing with those attributes. Maybe they are gone after reprojection.

Reply:
please give me a hint how to do that. I could add test with an identity projection and use assert_eq on both collections.

Reply:
Just access the field and check for that separately. You don't need to equal the whole collection.

Reply:
84054c1